plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java

```diff
@@ -61,8 +62,11 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
             case SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_EQUALITY:
             case SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY:
                 return false;
-
             case SUPPORTS_PREDICATE_EXPRESSION_PUSHDOWN:
+                // TODO remove once super has this set to true
+                verify(!super.hasBehavior(connectorBehavior));
+                return true;
```

Comment:
Mention SQL Server in the "Verify expression pushdown isn't expected by default" commit message